src/templates/drizzle/labelheader.hbs

```diff
@@ -0,0 +1,24 @@
+<{{tag}} {{#if id}}id="{{id}}"{{/if}} class="
+  drizzle-u-inlineBlock
+  drizzle-u-alignMiddle
+  drizzle-u-textNoWrap
+  drizzle-u-padRight">
+  {{#if key}}
```

Comment:
@mrgerardorodriguez 

Supplying an `id` lets us know to wrap things with a self-referencing "hidden" anchor. If we need to have an arbitrary `<a>` instead, we can omit the `id` and simply include whatever markup is needed via `@partial-block`:

```hbs
{{#> drizzle.labelheader tag="h3"}}
  <a href="./{{@key}}.html">{{data.title}}</a>
{{/drizzle.labelheader}}
```